kamelets/salesforce-delete-sink.kamelet.yaml

```diff
@@ -0,0 +1,107 @@
+    camel.apache.org/provider: "Apache Software Foundation"
+    camel.apache.org/kamelet.group: "Salesforce"
+  labels:
+    camel.apache.org/kamelet.type: "sink"
+spec:
+  definition:
+    title: "Salesforce Delete Sink"
+    description: "Removes an object from Salesforce."
+    required:
+      - sObjectId
+      - sObjectName
+      - clientId
+      - clientSecret
+      - userName
+      - password
+    type: object
+    properties:
+      sObjectName:
+        title: Object Name
+        description: Type of the object
+        type: string
+        example: Contact
+      sObjectId:
```

Comment:
I wonder if having to set the `sObjectId` as a parameter is right because that would mean that there is a 1-1 relation between the Kamelet and the object to delete (unless a `toD` is used but then it would make the integration a template of templates)

Reply:
So if it is not sent as parameter... extract it from the body?

Reply:
That may be an option
I don't know if you want to make both the `sObjectType` and `sObjectId` something you expect in the body

Reply:
That's also an option. Let me try :)

Reply:
Done.